addons/hr/static/tests/m2x_avatar_employee.test.js

```diff
@@ -1,15 +1,31 @@
 import { defineHrModels } from "@hr/../tests/hr_test_helpers";
 import { start } from "@mail/../tests/mail_test_helpers";
-import { describe, expect, test } from "@odoo/hoot";
+import { describe, expect, test, beforeEach } from "@odoo/hoot";
 import { waitFor } from "@odoo/hoot-dom";
 import { contains, makeMockServer, mountView, onRpc } from "@web/../tests/web_test_helpers";
 import { getOrigin } from "@web/core/utils/urls";
 
 describe.current.tags("desktop");
 defineHrModels();
+let env
+beforeEach(async () => {
+    const server = await makeMockServer();
+    env = server.env;
+    onRpc("resource.resource", "get_avatar_card_data", (params) => {
+        const resourceIdArray = params.args[0];
+        const resourceId = resourceIdArray[0];
+        const resources = env['hr.employee.public'].search_read([['id', '=', resourceId]]);
+        const result = resources.map(resource => ({
+            name: resource.name,
+            work_email: resource.work_email,
+            phone: resource.phone,
+            user_id: resource.user_id,
+        }));
+        return result;
+    });
+});
```

Comment:
It's best to define code in mock server (according to what is done in python) so that all tests can benefit from it rather than a single test suite.

Reply:
ah indeed, we could do that.